articles/search/search-get-started-vector.md

```diff
@@ -286,18 +291,76 @@ api-key: {{admin-api-key}}
             -0.00086512347
         ],
         "fields": "contentVector",
+        "select": "title, content, category"
         "k": 10
     },
-    "select": "title, category",
     "filter": "category eq 'Databases'"
 }
 ```
 
+
+### Cross-field vector search
+Cross-field vector search allows you to send a single query across multiple vector fields in your vector index. For this example, I want to calculate the similarity across both `titleVector` and `contentVector`:
```

Comment:
might want to add a comment that the scores are RRF scores now, not similarity scores. Which would explain why the values are very low.